### PR TITLE
api: update es to v7

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

File: inspire_matcher/api.py [hunk 1]
```diff
@@ -58,11 +59,17 @@ def match(record, config=None):
         config = current_app.config['MATCHER_DEFAULT_CONFIGURATION']
 
     try:
-        algorithm, doc_type, index = config['algorithm'], config['doc_type'], config['index']
+        algorithm = config['algorithm']
+        if ES_VERSION[0] == 7:
```
Comment (drjova) [inline inspire_matcher/api.py]: shouldn't be ``>=``

Comment (pazembrz) [reply inspire_matcher/api.py]: Yup it might be >= (let's hope they won't change their mind on version 8 :P )

File: inspire_matcher/api.py [hunk 2]
```diff
@@ -58,11 +59,17 @@ def match(record, config=None):
         config = current_app.config['MATCHER_DEFAULT_CONFIGURATION']
 
     try:
-        algorithm, doc_type, index = config['algorithm'], config['doc_type'], config['index']
+        algorithm = config['algorithm']
```
Comment (drjova) [inline inspire_matcher/api.py]: I think is not used anywhere

Comment (pazembrz) [reply inspire_matcher/api.py]: function `match` from here is used in `workflows.tasks.matching.py` in Next

**Pull request merged**